app/command_test.go

```diff
@@ -195,7 +198,12 @@ func TestHandleCommandResponsePost(t *testing.T) {
 
 	post, err = th.App.HandleCommandResponsePost(command, args, resp, builtIn)
 	assert.Nil(t, err)
-	assert.Equal(t, "@here", resp.Attachments[0].Text)
+	assert.Equal(t, "@channel", post.Message)
+	assert.Equal(t, 1, len(post.Attachments()))
+	for _, attachment := range post.Attachments() {
```

Comment:
nit: if we know the length is `1` by the assertion above, do we need the `range`? Could we just keep the original check?

Reply:
I think that would lead to a panic since we are using `assert` and following through. I suggest we use `require` if we remove this loop.

Reply:
how can I check if the post object have the attachment we are expecting without the range thing?

Reply:
I was imagining something like:
```go
	if assert.Len(t, post.Attachments(), 1) {
	    assert.Equal(t, "@here", post.Attachments()[0].Text)	
        }
```

Using `require` would also work, but it would stop the rest of the tests altogether too. If this was broken up into subtests, I'd definitely prefer `require`.

Reply:
got it! will update